src/strands/models/bedrock.py

```diff
@@ -321,7 +334,17 @@ def _format_bedrock_messages(self, messages: Messages) -> list[dict[str, Any]]:
         filtered_unknown_members = False
         dropped_deepseek_reasoning_content = False
 
-        for message in messages:
+        # Find the index of the last user message if wrapping is enabled
+        last_user_idx = -1
+        if guardrail_last_turn_only and messages:
```

Comment:
nit: Do we need to check messages here?
```suggestion
        if guardrail_last_turn_only:
```